### PR TITLE
Payment card

### PR DESCRIPTION
**Pull request by rakeshrkamath**

### CODE REVIEW

File: src/components/common/interfaces.tsx [hunk 1]
```diff
@@ -53,7 +53,11 @@ export interface foodItemProps {
 export interface foodcardDetailsProps {
   [key: string]: foodItemProps[];
 }
-
+export interface PaymentCardDetailsProps {
+  cardNumber: string;
```
Comment (naikashok) [inline src/components/common/interfaces.tsx:57]: cardNumber should be number right ?

File: src/components/common/paymentCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+} from "../../../variables";
+import { PaymentCardDetailsProps } from "../interfaces";
+const PaymentCard: FC<{ cardDetails: PaymentCardDetailsProps }> = ({
+  cardDetails,
+}) => {
+  const [primaryCheck, setPrimaryCheck] = useState(cardDetails.primaryStatus);
+  return (
+    <CardContainer>
+      <TickBox onClick={() => setPrimaryCheck(!primaryCheck)}>
+        <img
```
Comment (naikashok) [inline src/components/common/paymentCard/index.tsx]: Create common Image component and use it in all places.

File: src/components/common/paymentCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+          src={primaryCheck === true ? Checked : Unchecked}
+          className="checked"
+        ></img>
+        <PrimaryLabel
+          content={data.primary}
+          className={`${primaryCheck?.toString()}`}
+        />
+      </TickBox>
+      <CardDetails>
+        <VisaLogo>
```
Comment (naikashok) [inline src/components/common/paymentCard/index.tsx:36]: Same Create Image component

File: src/components/common/interfaces.tsx (hunk 1 above)
Comment (rakeshrkamath) [reply src/components/common/interfaces.tsx:57]: It also contains letters "2345 XXXX XXXX 6574"

File: src/components/common/paymentCard/index.tsx (hunk 2 above)
Comment (rakeshrkamath) [reply src/components/common/paymentCard/index.tsx:24]: Created common component

File: src/components/common/paymentCard/index.tsx (hunk 3 above)
Comment (rakeshrkamath) [reply src/components/common/paymentCard/index.tsx:34]: yes done

File: src/components/common/interfaces.tsx (hunk 1 above)
Comment (rakeshrkamath) [reply src/components/common/interfaces.tsx:57]: it does not contain only numbers "2345 XXXX XXXX 6574"

File: src/components/common/paymentCard/index.tsx (hunk 2 above)
Comment (rakeshrkamath) [reply src/components/common/paymentCard/index.tsx]: created

File: src/components/common/paymentCard/index.tsx (hunk 3 above)
Comment (rakeshrkamath) [reply src/components/common/paymentCard/index.tsx:36]: yes done

File: src/components/common/paymentCard/index.tsx (hunk 2 above)
Comment (naikashok) [reply src/components/common/paymentCard/index.tsx]: Still using  here ?

**Review by naikashok — APPROVED** [review]

**Pull request merged**